components/Nav.jsx

```diff
@@ -1,97 +1,100 @@
-import Link from 'next/link';
-import { useState } from 'react';
-import { Avatar, Image } from 'antd';
-import { Icon, InlineIcon } from '@iconify/react';
-import bxNotification from '@iconify/icons-bx/bx-notification';
+import Link from "next/link";
+import { useState } from "react";
+import { Avatar, Image } from "antd";
+import { Icon, InlineIcon } from "@iconify/react";
+import bxNotification from "@iconify/icons-bx/bx-notification";
+import firebase from "firebase/app";
+import Button from "@material-ui/core/Button";
 
 // This  Nav bar is referred to from https://dev.to/andrewespejo/how-to-design-a-simple-and-beautiful-navbar-using-nextjs-and-tailwindcss-26p1
-export const Nav = () => {
-    const [active, setActive] = useState(false);
-    const [openNotification, setOpenNotification] = useState(null);
+export const Nav = ({ email, signOut }) => {
```

Comment:
Nav now takes in props to be able to handle firebase authentication (specifically signing out)